index.html

```diff
@@ -0,0 +1,738 @@
+
+<body>
+  <div class="fixed-header">
+    <nav class="nav">
+      <div class="nav-logo">
+        <a href="#">
+          <img src="./images/tnw-logo.svg" alt="tnw logo">
+        </a>
+      </div>
+      <div class="nav-links-ipad-wrapper">
+        <div class="nav-bar-ipad-1">
+          <ul>
+            <li class="first-li-item">News</li>
+            <li>TNW2020</li>
+            <li>Business ▾</li>
+            <li>AMAs</li>
+            <li>Sapces</li>
+            <li>Terms and condintions</li>
+          </ul>
+          <ul class="right-lp-dsktp">
+            <li>About</li>
+            <li>Advertise</li>
+            <li>Jobs</li>
+            <li>Contact</li>
+          </ul>
+        </div>
+        <div class="nav-bar-ipad-2">
+          <ul>
+            <li>latest</li>
+            <li>Hard fork</li>
+            <li>plugged</li>
+            <li>fundamentals</li>
+            <li>readme</li>
+            <li class="desktop-only">growth quarters</li>
+            <li class="desktop-only">neural</li>
+          </ul>
+          <div class="search-i-wrapper">
+            <i class="material-icons nvb-search">search</i>
+          </div>
+        </div>
+      </div>
+      <ul class="nav-icons">
+        <li>
+          <a href="#">
+            <i class="material-icons nav-search">search</i>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <i class="material-icons nav-menu-icon">menu</i>
+          </a>
+        </li>
+      </ul>
+    </nav>
+    <div class="tech-fest-cta">
+      <a href="#">
+        Did you know we have a TECH FESTIVAL? Check it out →
+      </a>
+    </div>
+  </div>
+  <div class="top-articles">
+    <article class="top1">
+      <div class="cover-logo"></div>
+      <div class="cover-date">Friday - March 13, 2020</div>
+      <p>PLAY</p>
+      <h2>How to check if your PC can handle the latest games</h2>
+      <div class="ta-atcl-credits">
+        <p>by <a href="#">Rachel Kaser</a></p>
+        <p class="icon-adjust-flex">
+          <i class="material-icons">access_time</i>
+          1 hour ago
+          <i class="material-icons">share</i>2
+        </p>
+      </div>
+    </article>
+    <article class="top2">
+      <h2>Reddit, PayPal, Spotify, and more are speaking at TNW’s
+        tech festival in Amsterdam</h2>
+    </article>
+    <article class="top3">
+      <h2>This algorithm could improve emergency
+        responses by removing lies spread on Twitter</h2>
+      <div class="ta-atcl-credits">
+        <p>by <a href="#">Thomas Macaulay</a></p>
+        <p class="icon-adjust-flex">
+          <i class="material-icons">access_time</i>
+          2 hours ago
+          <span class="share-icon"><i class="material-icons ">share</i> 3</span>
+        </p>
+      </div>
+    </article>
+  </div>
+  <div class="love-wrapper">
+    <section class="latest-news">
+      <h2 class="ln-heading">Latest news <i class="material-icons">keyboard_arrow_right</i></h2>
```

Comment:
The articles in the latest news section are not showing the images. Check your links.